CFC_DataCollector/recommender/augment_trips_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import get_trips
+#import Profiles
+
+def augment_trips(trip_list):
+    '''
+    store trips to database
```

Comment:
Do we want to specify upfront that the data needs to be stored? Or should we allow the writers of that module (the API team) to determine how/when to store the data? They just need to return us valid alternates - we don't care about the other details.